Library/Formula/gst-plugins-good.rb

```diff
@@ -50,6 +50,15 @@ class GstPluginsGood < Formula
 
   depends_on "libogg" if build.with? "flac"
 
+  if MacOS.version < :lion
+    # Snow Leopard and below don't have strnlen()
+    # https://bugzilla.gnome.org/show_bug.cgi?id=756154
+    patch do
+      url "http://cgit.freedesktop.org/gstreamer/gst-plugins-good/patch/?id=fc203a4bd7eb1cecc0e17bcb7ec67e0672806867"
```

Comment:
It's also available from the [Bugzilla](https://bugzilla.gnome.org/attachment.cgi?id=313403&action=diff&context=patch&collapsed=&headers=1&format=raw); would Homebrew/patches still be preferable?